pwndbg/commands/next.py

```diff
@@ -53,7 +53,7 @@ def next_syscall(*args):
     """
     Breaks at the next syscall.
     """
-    while not pwndbg.next.break_next_interrupt() and pwndbg.next.break_next_branch():
+    while pwndbg.proc.alive and not pwndbg.next.break_next_interrupt() and pwndbg.next.break_next_branch():
```

Comment:
no, for me i got a big ugly stacktrace yelling at me that i can't continue because the process has exited already. so it was "continuing until the next syscall" but in the meanwhile the program gracefully terminated, ultimately throwing an exception at me